.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,77 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "1.5.0" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "1.5.0" ]
+  schedule:
+    - cron: '19 23 * * 0'
```

Comment:
remove this?   do we need this schedule to run?  If so, why running on At 23:19 on Sunday?